src/ontology/metadata/icd11.yml

```diff
@@ -0,0 +1,13 @@
+id: ICD11
+label: International Classification of Diseases 11th Revision
+prefix_map:
+  ICD11: http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/ICD11/
```

Comment:
does not seem correct as per our conversation on ICD11 prefices

Reply:
This is an artefact of this being branched off of #412, which is still WIP!

Reply:
Resolved now following `rebase`.